handler/payloadLatestReports.js

```diff
@@ -36,7 +36,7 @@ export default async function(chat) {
             translateReport(report, chat).text,
             [ buttonPostback(
                 chat.getTranslation(translations.readReport),
-                { action: 'report', report: report.id }
+                { action: 'report', report: report.id, 'language': chat.language }
```

Comment:
why is this needed?

Reply:
Since not every report exists in every language, you can get a list of all reports with language A, change settings to language B, go back to the open list of reports and open one report with no translation in B and receive an error.
Figured this out while testing the different lists we use.

Reply:
ohh that makes sense, thanks :)